### DISCUSSION TITLE
为什么我一直打不开readme里的文档页面

### DISCUSSION THREAD

**Discussion by ouer1972** [rbatis/rbatis#264] [General]
就卡在这几个上
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/vue.css" />
<script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
<script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-rust.min.js"></script>
<script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-toml.min.js"></script>
<script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-json.min.js"></script>

**Comment by zhuxiujia** [comment]
> 就卡在这几个上
> 
> <script src="//cdn.jsdelivr.net/npm/docsify@4"></script> <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-rust.min.js"></script> <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-toml.min.js"></script> <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-json.min.js"></script>

科学上网，懂的都懂
